api/src/main/java/org/eclipse/microprofile/opentracing/Trace.java

```diff
@@ -36,7 +35,6 @@
  */
 
 @Documented
-@InterceptorBinding
```

Comment:
Steve, 
Although previously I said you can add the InterceptorBinding via cdi extension, it is best to make the annotation clearly specify this. With the presence of the interceptorBinding, it means this annotation cannot be added to a final class. 
Keeping it in the code is better.